tooltester/src/main/java/io/dockstore/tooltester/client/cli/Client.java

```diff
@@ -419,30 +373,57 @@ void countNumberOfTests(Tool verifiedTool) throws ApiException {
         SourceFile testParameter;
         List<SourceFile> testParameterFiles;
         DockstoreTool dockstoreTool;
+        Workflow workflow;
         Long containerId;
-        dockstoreTool = containersApi.getPublishedContainerByToolPath(verifiedTool.getId());
-        containerId = dockstoreTool.getId();
-        for (ToolVersion version : verifiedTool.getVersions()) {
-            String tag = version.getName();
-            dockerfile = containersApi.dockerfile(containerId, tag);
-            for (ToolVersion.DescriptorTypeEnum descriptorType : version.getDescriptorType()) {
-                switch (descriptorType.toString()) {
-                case "CWL":
-                    descriptor = containersApi.cwl(containerId, tag);
-                    break;
-                case "WDL":
-                    descriptor = containersApi.wdl(containerId, tag);
-                    break;
-                default:
-                    break;
+        if (verifiedTool.getToolclass().getName().equals("Workflow")) {
+            workflow = workflowsApi.getPublishedWorkflowByPath(verifiedTool.getId().replace("#workflow/", ""));
+            containerId = workflow.getId();
+            for (ToolVersion version : verifiedTool.getVersions()) {
+                String tag = version.getName();
+                for (ToolVersion.DescriptorTypeEnum descriptorType : version.getDescriptorType()) {
+                    switch (descriptorType.toString()) {
+                    case "CWL":
```

Comment:
Is there an enum for this?

Reply:
Encounters "Constant Expression Required" when trying with `case ToolVersion.DescriptorTypeEnum.CWL.toString()`  I think it'd only work if the thing inside switch is also an enum.